templates/projects/hello-world/files/common/models/note.js

```diff
@@ -1,8 +0,0 @@
-module.exports = function(Note) {
-  Note.greet = function(msg, cb) {
-    process.nextTick(function() {
-      msg = msg || 'world';
-      cb(null, 'Hello ' + msg);
-    });
-  };
```

Comment:
AFAICT, the greet method was removed completely. Instead, it should be added to the `Message` model.

https://github.com/strongloop/loopback-workspace/blob/0e6b3c2a5f916011f5144c4352cacba18a7c1b3a/templates/projects/hello-world/files/common/models/message.js

And the remoting metadata should go here:
https://github.com/strongloop/loopback-workspace/blob/0e6b3c2a5f916011f5144c4352cacba18a7c1b3a/templates/projects/hello-world/files/common/models/message.json